lib/scoreboard.rb

```diff
@@ -0,0 +1,82 @@
+class Scoreboard
+
+  attr_reader :score, :frame_score, :frames
+
+  def initialize
+    @frames = 1
+    @frame_score = {}
+    @score = []
+  end
+
+  def run
+    frames_to_9
+    frame_10
+    adding_extra_points
+    puts "Your total score is #{total}"
+  end
+
+  def adding_extra_points
+    adding_strike_points
+    adding_spare_points
+    p @frame_score
+  end
+
+  def frames_to_9
+    while @frames < 10
+      first_roll
+      strike? ? @frames += 1 : second_roll
+    end
+  end
+
+  def frame_10
+    first_roll
+    if strike?
+      bonus_roll
```

Comment:
I think this means the 10th frame will have 3 rolls if the first is a strike? I think if the first is a strike and the second isn't, it should only have two rolls. (Not 100% sure though!)